Resources/Prototypes/_Nuclear14/Entities/Objects/Misc/tiles.yml

```
@@ -0,0 +1,450 @@
+- type: entity
+  name: metal floor blue
+  parent: FloorTileItemBase
+  id: FloorTileItemMetalBlue
+  components:
+  - type: Sprite
+    state: steel
+  - type: Item
+    heldPrefix: steel
+  - type: FloorTile
+    outputs:
+      - Plating
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)